bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -0,0 +1,99 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Firefox Quantum for Enterprise Web Browser | Firefox')}}{% endblock %}
+{% block page_desc %}{{_('A managed version of the super-fast new Firefox.')}}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_enterprise' %}
+{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+  <main role="main">
+  <section class="intro content-section">
+    <div class="content">
+      <div class="copy">
+        <img class="logo" src="{{ static('img/logos/firefox/logo-quantum.png') }}" width="100" height="100" alt="" />
+        {% block head_content %}
```

Comment:
It appears that many of the `block`s in the file aren't necessary (and are likely carryovers from a different template). If the block isn't overriding a parent template block, and won't be overridden in a child template, it should be removed.